lib/mods/replace-methods.js

```diff
@@ -65,42 +65,42 @@ function getArgumentString(functionCall) {
 function replaceGlobalMethodCalls(node, imports) {
   if (nodeTypes.isCallExpression(node) &&
       isGlobal(node.getExpression())) {
-    let mod; let args; let type; let callee;
+    let moduleName; let args; let type; let callee; let propertyName;
 
     // property access methods
     if (nodeTypes.isPropertyAccessExpression(node.getExpression())) {
       const propertyAccess = node.getExpression();
-      name = propertyAccess.getSymbol().getEscapedName();
+      propertyName = propertyAccess.getName();
       type = 'method';
 
       let baseType = propertyAccess.getExpression().getType();
       const baseConstructSignature = baseType.getConstructSignatures()[0];
 
+      // the method is static if it has a constructSignature
       if (baseConstructSignature) {
         baseType = baseConstructSignature.getReturnType();
         type = 'static';
```

Comment:
Move the `type` assignment to be the first line inside the `if` block, for symmetry, since that's what you do in the other two cases. (And since it's kind of the "most important" thing to set.)